presto-main/src/test/java/com/facebook/presto/operator/aggregation/reservoirsample/TestWeightedDoubleReservoirSample.java

```diff
@@ -58,6 +60,7 @@ public void testMany()
 
         long streamLength = 1_000_000;
         for (int i = 0; i < streamLength; ++i) {
+            assertEquals(reservoir.getTotalPopulationWeight(), (double) i);
```

Comment:
nit: you shouldn't need to cast int to double.  Java should do it automatically.

Reply:
Just a point of interest: it passes with this cast, and fails without it. Bypassed it with a tolerance parameter (which is more correct here in any case).